changelog.md

```diff
@@ -6,6 +6,40 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### 0.41.0
+
+### Added
+
+* `state init` is now a stable command, meaning you no longer need to opt-in to
+  unstable commands to use it.
+* Signing up for a new account now opens the account creation page in your
+  browser.
```

Comment:
```suggestion
* Signing up for a new account now opens the account creation page in your
  browser. Instead of bringing you to the sign in page.
```

Reply:
This should be in Changed imo. It always opened your browser, the URL is just different now (brings you straight to signup).